demos/ros_examples/package.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>ros_examples</name>
+  <version>0.0.0</version>
+  <description>Control libraries demos included in a ROS package</description>
+  <maintainer email="baptiste.busch@epfl.ch">root</maintainer>
```

Comment:
what is `root` doing here?

Reply:
default value I guess I didn't check that